kubernetes_deploy/staging/deployment.yml

```diff
@@ -25,3 +25,28 @@ spec:
             secretKeyRef:
               name: secret-key
               key: SECRET_KEY
+        - name: SENTRY_DSN
+          valueFrom:
+            secretKeyRef:
+              name: sentry
+              key: dsn
+        - name: DB_USERNAME
```

Comment:
🙋‍♂ What do you think about collapsing these four environment variables into one? We could use the [URL configuration](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING): `postgresql://[user[:password]@][netloc][:port][/dbname][?param1=value1&...]`, for example:

```
postgresql://user:secret@host:5432/dbname
```

Reply:
Very happy with that as a plan. Django itself doesn't take the connection string as a setting (you have to pass it the various elements in a dict), so this'll require adding `dj-database-url` (or `django-environ`, which provides it and some other related packages)
Should be a small amount of work still, but another ticket